client/src/js/screens/pages/SingleRecipe/components/CreateReview.js

```diff
@@ -19,6 +19,10 @@ export default class CreateReview extends Component {
         recipeId: this.props.params.id,
         review: this.state.review
       });
+
+      this.setState({
+        review: ''
+      });
     } catch (e) {
```

Comment:
What is e? e is not an explicit name for what this is supposed to do. 
 
For eg: Is it an `error` or an `event`?

Please make sure your variable name is explicit enough as to what they are supposed to be doing